srcpkgs/libyang/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'libyang'
+pkgname=libyang
+version=0.16r3
+revision=1
+_ver="0.16-r3"
```

Comment:
Better to have version defined once, please change like here: https://github.com/void-linux/void-packages/blob/561b3db2509f0ec6807819361d4c5fbe5230d1cb/srcpkgs/android-tools/template#L8